common/src/main/java/com/microsoft/identity/common/internal/controllers/CommandDispatcher.java

```diff
@@ -166,10 +169,15 @@ public static void submitSilent(@SuppressWarnings(WarningType.rawtype_warning) @
     public static FinalizableResultFuture<CommandResult> submitSilentReturningFuture(@SuppressWarnings(WarningType.rawtype_warning) @NonNull final BaseCommand command) {
 
         final String methodName = ":submitSilent";
-        Logger.verbose(
-                TAG + methodName,
-                "Beginning execution of silent command."
-        );
+
+        final CommandParameters commandParameters = command.getParameters();
+        final String correlationId = initializeDiagnosticContext(commandParameters.getCorrelationId(),
+                commandParameters.getSdkType() == null ? SdkType.UNKNOWN.getProductName() : commandParameters.getSdkType().getProductName(), commandParameters.getSdkVersion());
```

Comment:
Style nit: can we format this line to get it down to a line width < 100 char?

(I see there's a few places within this PR where lines get wide, consider breaking these lines up for readability)